packages/utils/src/debounce/debounce.ts

```diff
@@ -6,7 +6,7 @@ import awesomeDebouncePromise from 'awesome-debounce-promise';
  * @param {number} [time = 800] delay in ms
  * @param {Object} [config = {onlyResolvesLast: true}] <a target="_blank" href="https://github.com/slorber/awesome-debounce-promise#options">more details</a>
  */
-export const debounceFunction = (asyncFunction, time = 800, config = { onlyResolvesLast: true }) =>
+export const debounceFunction = (asyncFunction: (...args: unknown[]) => unknown, time = 800, config = { onlyResolvesLast: true }) =>
```

Comment:
Can you make it generic? So devs can pass async function type?

Reply:
pushed the change, thanks @Hyperkid123